docs/Guide/1.md

```diff
@@ -0,0 +1,139 @@
+---
```

Comment:
JS guide goes in js.waku.guide. not here in waku.guide. waku.guide is implementation agnostic.

Reply:
Also, I will update the js guides myself as I have done a lot of changes so don't worry about that :)